lib/mobility-core/src/Kernel/External/Verification/Idfy/Types/Response.hs

```diff
@@ -33,7 +33,7 @@ type VerificationResponse = IdfyResponse IdfyResult
 
 type VerificationResponseList = [IdfyResponse IdfyResult]
 
-type IdfyResult = Output DLVerificationOutput RCVerificationOutput
+type IdfyResult = Output DLVerificationOutput VT.RCVerificationResponse
```

Comment:
This change will break the Idfy webhook. We need to keep the `RCVerificationOutput` to decode the idfy webhook resposne and then convert it into `VT.RCVerificationResponse` while sending it to Domain.